### ISSUE TITLE
Sidebar header text and labels do not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52586]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Windows 10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to View(Alt+V) menu option and select "Explorer" from dropdown.
3.Check the color contrast ratio of the content on the page i.e. Open Editors ,1 UNSAVED,No Folder Opened etc.

**Actual**
Color contrast ratio is less than 4.5:1 for the content present on the page.i.e. Open Editors ,1 UNSAVED,No Folder Opened etc.

**Expected:**
Color contrast ratio should be more than 4.5:1 for the content present on the page.i.e. Open Editors ,1 UNSAVED,No Folder Opened etc.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The End user will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

Attachment For Reference:
![accessibility a11y_vscode_viewexplorer_colorcontrast](https://user-images.githubusercontent.com/39204909/41713965-6b67f4bc-756c-11e8-95f6-688b4e82f687.png)
![accessibility a11y_vscode_viewexplorer_colorcontrast_1](https://user-images.githubusercontent.com/39204909/41713966-6b94c294-756c-11e8-80e2-517512a39757.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
This only fails on our light theme
